data/ui/figure_settings.blp

```diff
@@ -41,86 +41,81 @@ template $GraphsFigureSettingsWindow : Adw.PreferencesWindow {
 
     Adw.PreferencesGroup {
       title: _("Axis Limits");
-      Adw.ActionRow no_data_message {
-        title: _("No Data");
-        subtitle: _("Add data to the figure to set axis limits");
+    	Adw.PreferencesGroup no_data_message {
+        Adw.ActionRow {
+          title: _("No Data");
+          subtitle: _("Add data to the figure to set axis limits");
+        }
       }
       Box {
-      	orientation: horizontal;
-      	margin-bottom: 12;
-      	Adw.PreferencesGroup {
-      	  margin-end: 3;
-      	  Adw.EntryRow min_left {
-	          title: _("Minimum Left-Hand Y-Axis");
-	          visible: false;
-	        }
+        orientation: vertical;
+        spacing: 12;
+        visible: bind no_data_message.visible inverted;
+        Box left_limits {
+        	orientation: horizontal;
```

Comment:
The indentation is all over the place here, perhaps your builder config is set to tabs instead of spaces?

Reply:
That's so weird, it looked all fine in Builder. But double checking it, this commit has a random mix of spaces and tabs. Not sure why that happened, as I double checked and it is set to spaces. Should be OK though.